*** HgProto.ts	2022-07-13 22:50:53.000000000 -0700
--- HgProto-patched.ts	2022-07-13 22:50:48.000000000 -0700
***************
*** 3,9 ****
   * compiler version: 3.15.8
   * source: src/proto/HgProto.proto
   * git: https://github.com/thesayyn/protoc-gen-ts */
! import * as dependency_1 from "./../../node_modules/@thegraid/wspbclient/lib/proto/GgProto";
  import * as pb_1 from "google-protobuf";
  export enum HgType {
      hg_none = 0,
--- 3,9 ----
   * compiler version: 3.15.8
   * source: src/proto/HgProto.proto
   * git: https://github.com/thesayyn/protoc-gen-ts */
! import * as dependency_1 from "@thegraid/wspbclient";
  import * as pb_1 from "google-protobuf";
  export enum HgType {
      hg_none = 0,
*************** export enum hgField {
*** 29,34 ****
--- 29,43 ----
  }
  export class HgMsgBase extends pb_1.Message {
      #one_of_decls: number[][] = [];
+     get has_type() { return pb_1.Message.getField(this, hgField.hgField_type) }
+     get has_client() { return pb_1.Message.getField(this, hgField.hgField_client) }
+     get has_player() { return pb_1.Message.getField(this, hgField.hgField_player) }
+     get has_name() { return pb_1.Message.getField(this, hgField.hgField_name) }
+     get has_json() { return pb_1.Message.getField(this, hgField.hgField_json) }
+     get has_inform() { return pb_1.Message.getField(this, hgField.hgField_inform) }
+     get has_roster() { return this.roster?.length > 0 }
+     get has_client_to() { return pb_1.Message.getField(this, hgField.hgField_client_to) }
+ 
      constructor(data?: any[] | {
          type?: HgType;
          client?: number;
